### PR TITLE
Navigation bar

### PR DESCRIPTION
**Pull request by BenjaminNeustadt**
css draft 1

### CODE REVIEW

File: views/layout.hbs [hunk 1]
```diff
@@ -1,13 +1,42 @@
...
+  </a></li>
+
+   <li><a><form action='/sessions?_method=DELETE' method='POST'>
+    <input class="nav-buttons" type="submit" value="Log Out">
+  </form>
+  </a></li>
+
+</ul>
+
     <title>{{title}}</title>
```
Comment (sophiegilder) [inline views/layout.hbs:34]: This should maybe be hard coded

**Review by sophiegilder — CHANGES_REQUESTED** [review]

File: views/layout.hbs [hunk 2]
```diff
@@ -1,13 +1,42 @@
 <!DOCTYPE html>
 <html>
   <head>
+  <ul>
+
```
[the remark this answers is not in the document]
Comment (BenjaminNeustadt) [reply views/layout.hbs:9]: resolved

File: views/posts/index.hbs [hunk 3]
```diff
@@ -1,10 +1,10 @@
 <h1>Timeline</h1>
 <ul class="posts">
   {{#each posts}}
-
-  <li><div class="post-text">{{this.message}}, {{this.firstname}}</div>
+  <div class="post"
```
Comment (sophiegilder) [inline views/posts/index.hbs:4]: can you close this tag, and close the div

**Review by sophiegilder — CHANGES_REQUESTED** [review]

**Review by sophiegilder — APPROVED** [review]

**Review by abacus2020 — APPROVED** [review]
Great work! So nice to have more visual elements on the page + good consistency with f<>acebook

**Pull request merged**